docs/source/gbd2017_models/causes/neoplasms/stomach_cancer/cancer_model.rst

```diff
@@ -0,0 +1,456 @@
+     - Source 
+     - Sink 
+     - Value
+     - Notes
+   * - i_pc
+     - S
+     - PC
+     - incidence_c414 / prevalence_S
+     -    
+   * - i_c
+     - PC
+     - C
+     - 1/MST per person-year
+     - See MST definition in table below
+   * - r
+     - C
+     - R
+     - 0.1 per person-year for each sex and age group	
+     - To be consistent with 10 year GBD assumption
+
+.. list-table:: Data Sources
+   :widths: 20 25 25 25
+   :header-rows: 1
+   
+   * - Measure
+     - Sources
+     - Description
+     - Notes
+   * - prevalence_c414
+     - need to get filepath
+     - CSU stomach cancer prevalence forecasts
+     - 2020-2040; defined as proportion of population with condition
+   * - csmr_c414
+     - need to get filepath
+     - CSU stomach cancer cause specific mortality rate forecast
+     - 2020-2040; defined as deaths per person-year in general population
+   * - incidence_rate_c414
+     - need to get filepath
+     - CSU stomach cancer cause-specific mortality rate forecast
+     - 2020-2040; defined as incidence cases per person-year in general population
+   * - disability_weight_s{248, 249, 250, 251}
+     - YLD appendix
+     - Sequela disability weights
+     - 0.288 (0.193-0.145), 0.049 (0.031-0.072), 0.451 (0.307-0.6), 0.54 (0.377-0.687)
+   * - prevalence_s{248, 249, 250, 251}
+     - GBD 2019, COMO, decomp_step='step4'
+     - stomach cancer sequelae prevalence
+     - Not forecasted
+   * - MST
+     - 4 months (95% CI: ); ?? distrbution of uncertainty at draw level
+     - Mean sojourn time; duration of time between onset of the asymptomtic stomach cancer to the clinical phase
+     - See below for instructions on how to sample and research background. NOTE: may update this value
+   * - AST
+     - ? (95% CI: ?); normal distribution of uncertainty at the draw level
+     - Average survival time; mean duration of time between detection and death
+     - See details below for sampling below. PLACEHOLDER VALUE
+
+.. todo::
+
+	Update/confirm placeholder values
+
+Mean Sojourn Time
+^^^^^^^^^^^^^^^^^
+
+**Parameter for Use in Model:**
+
+This parameter should be sampled *at the draw level* from the distribution detailed below and should be applied universally to all simulants within that draw.
+
+.. code-block:: Python
+
+  from scipy.stats import norm
+
+  # mean and 0.975-quantile of normal distribution for mean difference (MD)
+  mean = ??
+  q_975 = ??
+
+  # 0.975-quantile of standard normal distribution (=1.96, approximately)
+  q_975_stdnorm = norm().ppf(0.975)
+
+  std = (q_975 - mean) / q_975_stdnorm # std dev of normal distribution
+
+  # Frozen normal distribution for MST, representing uncertainty in the parameter
+  mst_distribution = norm(mean, std)
+
+.. note::
+
+  May consider adding individual-level variation to this parameter at a later date.
+
+
+Average Survival Time
+^^^^^^^^^^^^^^^^^^^^^
+
+**Parameter for Use in Model:**
+
+.. warning::
+
+  This is currently a stand-in value
```

Comment:
I believe that there is a value of AST that is implied by the incidence and mortality rates from GBD

Reply:
Used in the lung cancer documentation:
1 / (amcr - emr + csmr)

Reply:
noted!